src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -414,6 +436,135 @@ private void LogMessage(string message, Exception e = null, string database = nu
             }
         }
 
+        private List<ResponsibleNodeInfo> GetResponsibleNodesForBackupTasks(RawDatabaseRecord rawRecord, string databaseName, DatabaseTopology topology, TransactionOperationContext context)
```

Comment:
move all this to a partial class `ClusterObserver.Backup` or something

Reply:
done